modules/user_accounts/php/edit_user.class.inc

```diff
@@ -1200,8 +1200,8 @@ class Edit_User extends \NDB_Form
         //======================================
         if ($editor->hasPermission('examiner_multisite')) {
             $matched = false;
-            foreach ($values as $k => $v) {
-                if (preg_match("/^ex_[0-9]+$/", $k)) {
+            foreach ($values as $v) {
+                if (preg_match("/^ex_[0-9]+$/", $v)) {
```

Comment:
I think it's the key we should be matching on here, not the value. the value is only `on` or NULL